### PR TITLE
[ui][fix] activity and metrics feature migration

### PR DESCRIPTION
**Pull request by mkralik3**
_**[ui][fix] activity feature migration**_
 * activity feature was migrated
 * basic filter was migrated
 * advanced filter was migrated
 * activity view was migrated

_**[ui][fix] metrics feature migration**_
* metrics feature was migrated
* metrics view was migrated
* fixed work with a time in the activity test

//skip-ci

### CODE REVIEW

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Connection - CRUD | Credentials | <font color="red"><b>FAILED</b></font> | mcada@redhat.com
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mkralik3** [comment]
It needs to update SimpleDateFormat for 12 hour format time

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Integration - Database | Smoke - Periodic invocation to Insert
Connection - CRUD | Credentials | <font color="red"><b>FAILED</b></font> | mcada@redhat.com
Connection - CRUD | Create and delete | <font color="red"><b>FAILED</b></font> | mcada@redhat.com
Connection - CRUD | Kebab menu | <font color="red"><b>FAILED</b></font> | mcada@redhat.com
Connection - CRUD | Kebab menu edit and view
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mkralik3** [comment]
blocked by: https://github.com/syndesisio/syndesis/issues/5892

File: ui-tests/src/test/java/io/syndesis/qe/steps/integrations/summary/ActivitySteps.java [hunk 1]
```diff
@@ -193,7 +195,10 @@ public void checkAllDurationOfStep(int indexActivity) {
         int numberOfRow = activityTab.getActivityLogRows(indexActivity - 1).size();
         for (int indexRow = 0; indexRow < numberOfRow; indexRow++) {
             Assertions.assertThat(activityTab.getColumnInRowInActivityLog(indexActivity - 1, indexRow, Activity.COLUMN.DURATION))
-                    .matches("^.*(\\d+ ms|\\d+ seconds)|(-)$");
+                .matches("^.*(\\d+ ms|\\d+ seconds)|(NaN)$"); // NaN is workaround for gh-5715
+            //          prepared reproducer for gh-5715
```
Comment (mcada) [inline ui-tests/src/test/java/io/syndesis/qe/steps/integrations/summary/ActivitySteps.java]: do you have a jira issue to track this task?

Comment (mkralik3) [reply ui-tests/src/test/java/io/syndesis/qe/steps/integrations/summary/ActivitySteps.java]: Yes, FUSEQE-4587 and FUSEQE-4588

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**